### PR TITLE
[NABI-14]  ImageUploader, integrating with cropping uploader

### PR DESCRIPTION
**Pull request by obetancourth**
Added Cropper Dependency , Cropper Wrapper developed to read a file (html file input) showed in a canvas and get base64 data representing a scaled cropped image in png format. Created integration with existing redux store for avatar element. Wrapper may recieve an url string to show actual avatar or a base64 repesentation of an image. Change method return a base64 image respesentation that can be uploaded and trasnformed into a binary image file on backend.

### CODE REVIEW

File: src/ts/ProfileBuilder/components/BasicInfo/CropprWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  }
+
+  render() {
+    let currentLogo = this.state.baseImage ;
+    let actions = null;
+    if ( this.state.isCropping ) {
+      actions = (<Button color="primary" onClick={this.handleCrop} variant="raised">Change Photo</Button>);
+    }
+    const imgStyle = ( !this.state.isCropping ) ? {
+                                                  width: '260px',
```
Comment (iamvane) [inline src/ts/ProfileBuilder/components/BasicInfo/CropprWrapper.tsx]: Quisiera que la el image uploader tuviera un width/height set to 140px, eliminate the  border and the padding.
See attachement.
image

File: src/ts/Registration/actions.ts [hunk 2]
```diff
@@ -6,10 +6,22 @@ interface CreateUser extends Action {
   user: UserState;
 }
 
+interface ChangeAvatar extends Action {
+  email: string;
```
Comment (iamvane) [inline src/ts/Registration/actions.ts]: Por curiosidad, por que pasas el email? lo pasas para que sirva en identificador unico?

**Comment by obetancourth** [comment]
Es corecto ya que no vi identificador y el indice al ver un arreglo no oude
suponer que era uno solo

**Comment by obetancourth** [comment]
En unas horas subo el cambio

**Pull request merged**